.github/workflows/pytest-miscellaneous-tests.yml

```diff
@@ -0,0 +1,102 @@
+---
+# This workflow is meant as a foundational workflow for running integration/unit tests on multiple targeted
+# ubuntu versions with multiple python versions.
+#
+# This workflow utilizes the build-dependency-cache workflow which sets up the environment dependencies using
+# bootstrap.py --all
+#
+
+# Documentation for the syntax of this file is located
+# https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+
+# The workflow name will show up in the action tab on github during execution
+# https://github.com/VOLTTRON/volttron/actions (or if you are pushing to your own fork change the user)
+name: Miscellaneous platform tests
+
+on:
+  push:
+    branches:
+    - develop
+    - releases/**
+  pull_request:
+    branches:
+    - main
+    - develop
+    - releases/**
+  
+jobs:
+  # The job named build
+  build:
+    # The strategy allows customization of the build and allows matrixing the version of os and software
+    # https://docs.github.com/en/free-pro-team@l.atest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy
+    strategy:
+      fail-fast: false
+      matrix:
+        # Each entry in the os and python-version matrix will be run 
+        os: [ ubuntu-20.04 ]
+        python-version: [ 3.8 ]
+    
+    # Run-on determines the operating system available to run on
+    # - At the current time there is only ubuntu machine 20.04 available
+    # - This uses the matrix os from the strategy above
+    runs-on: ${{ matrix.os }}
+
+    # Each step will be run in order of listing.
+    steps:
+      # Checkout the volttron repository and set current direectory to it
+      - uses: actions/checkout@v2
+
+      # Setup the python environment for the operating system
+      - name: Set up Python ${{matrix.os}} ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      # Run the specified tests and save the results to a unique file that can be archived for later analysis
+      - name: Run certs test on ${{ matrix.python-version }}, ${{ matrix.os }}
```

Comment:
Is there a reason we are specifying exactly the tests to run rather than running the directory of tests?

if test_path was volttrontesting/platform that would get all of the tests here and if there are any additional the script would automatically pick them up and test them.

Reply:
@bonicim had told me to only include those tests in /platform that are passing. There were several that were resulting in errors or failing that he said not to include, and we would work on getting those fixed in a different task. So rather than testing the whole directory, I just listed the ones that were passing.

Reply:
@gwenkidd Craig made a good point about what to do with the failing tests. Consequently, let's not only document the failing tests as Jira tickets but also update those failing tests using pytests markers: see the updated Jira ticket for details. More importantly, this will increase the scope of the Jira ticket and thus this PR. Please put this PR in draft mode see https://github.blog/changelog/2020-04-08-convert-pull-request-to-draft/; then mark it as ready for review when it's ready.
@gwenkidd Craig made a good point about what to do with the failing tests. Consequently, let's not only document the failing tests as Jira tickets but also update those failing tests using pytests markers: see the updated Jira ticket for details. More importantly, this will increase the scope of the Jira ticket and thus this PR. Please put this PR in draft mode see https://github.blog/changelog/2020-04-08-convert-pull-request-to-draft/; then mark it as ready for review when it's ready.

Reply:
@bonicim Will do!

Reply:
@bonicim All the failing/erroring tests have been marked with the pytest fail marker.